packit_service/service/models.py

```diff
@@ -145,3 +152,61 @@ def create(
         if save:
             b.save()
         return b
+
+
+def get_sa_session() -> Session:
+    """ get SQLAlchemy session """
+    global session_instance
+    if session_instance is None:
+        url = (
+            f"postgres+psycopg2://{os.getenv('POSTGRESQL_USER')}"
+            f":{os.getenv('POSTGRESQL_PASSWORD')}@postgres:5432/{os.getenv('POSTGRESQL_DATABASE')}"
+        )
+        engine = create_engine(url)
+        Session = sessionmaker(bind=engine)
+        session_instance = Session()
+    return session_instance
+
+
+# https://github.com/python/mypy/issues/2477#issuecomment-313984522 ^_^
+if TYPE_CHECKING:
+    Base = object
+else:
+    Base = declarative_base()
+
+
+class GitHubProject(Base):
+    __tablename__ = "github_projects"
+    id = Column(Integer, primary_key=True)
+    namespace = Column(String)
+    repo_name = Column(String)
+
+
+class PullRequest(Base):
+    __tablename__ = "pull_requests"
+    id = Column(Integer, primary_key=True)  # our database PK
+    pr_id = Column(
+        String, index=True
+    )  # GitHub PR ID - let's not make this PK since we can't control it
+    project_id = Column(Integer, ForeignKey("github_projects.id"))
+    project = relationship("GitHubProject")
+
+
+JOB_TYPE_SRPM = "SRPM"
+JOB_TYPE_COPR_RPM = "COPR-RPM"
+JOB_TYPE_TFT = "TFT"
```

Comment:
What about some enum -- will be probably useful elsewhere.

Reply:
as for enums:
* didn't dig into how database engines or SA deal with enums
* I hate when enums are represented as numbers in DB - what the F should know what 2 or 3 does stand for?
* I hope it wouldn't be a big problem to alter the enum in future
* actually I think I hate enums, they bring more problems than benefits

how about I just add your suggestion as a comment to the code?

Reply:
>     * I hate when enums are represented as numbers in DB - what the F should know what 2 or 3 does stand for?

You can use a string enum on the top of this. (So the values are strings, not ints.)

> how about I just add your suggestion as a comment to the code?

:+1: